app/src/services/dataService.tsx

```diff
@@ -29,6 +30,21 @@ export function fetchLocationData(inputValue: string, callback: (arg?: any) => {
     );
 }
 
+export function fetchDepartureAirport(inputValue: string, callback: (arg?: any) => void) {
+    const method = 'GET';
+    const endpoint = process.env.REACT_APP_TRINGO_API;
+    const url = inputValue ? endpoint!.concat('/api/v1/airports/GetAirportCoordinates?airportCode=').concat(inputValue)
+        : inputValue;
+    callApi(url, method).then(
+        (response:IDepartureAirport) => {
```

Comment:
I think better to use _Coordinates_ (from flightDestinationRequest.tsx) instead of IDepartureAirport

Reply:
sure I will make changes for this